test/test_nestedtensor.py

```diff
@@ -110,7 +110,7 @@ def test_2d_nested_tensor(self, batch_size, max_seq_len, vocab_size):
                 length = np.random.randint(low=1, high=max_seq_len)
             row = list(np.random.randint(low=0, high=vocab_size, size=(length,)))
             data.append(row)
-            nested_tensor_ref_list.append(torch.tensor(row))
+            nested_tensor_ref_list.append(torch.Tensor(row))
```

Comment:
not sure why tf this just started failing?

Reply:
thats weird and not good lol